### PR TITLE
Cleanups to the "Getting Started" section

### PR DESCRIPTION
**Pull request by potocnyj**
This isn't exhaustive, just some minor formatting and language fixes for the most part.

### CODE REVIEW

**Comment by potocnyj** [comment]
@Preetam mind taking a look?

File: content/getting-started/installing.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ section on the Managing Hosts page.
 
 ## Containerized
 
-In *containerized* monitoring, agents are configured to run in a LXC type virtualization environment such as those provided by Heroku (see [Installing For Heroku](#installing-for-heroku)). Given the nature of containerized resources, agents can only monitor using [Off-Host Monitoring](#off-host-monitoring) method.
+In *containerized* monitoring, agents are configured to run in a LXC type virtualization environment such as those provided by Heroku (see [Installing For Heroku](#installing-for-heroku)). Given the nature of containerized resources, agents can only monitor using [off-host monitoring](#off-host-monitoring) method.
```
Comment (Preetam) [inline content/getting-started/installing.md]: I think we support more than just LXC in the containerized mode (Docker).

File: content/getting-started/installing.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ The database-specific metrics agent needs a login to capture status metrics and
 	CREATE USER monitor@'%' IDENTIFIED BY 'somepass';
 	GRANT SELECT, PROCESS, REPLICATION CLIENT ON *.* TO monitor@'%';
 
-After creating this user, find the host on the Hosts List, and select it with the mouse, then click the gear icon that appears to the right of the host's name.
+After creating this user, find the host on the Hosts List, and click on the settings gear to the right:
```
Comment (Preetam) [inline content/getting-started/installing.md]: I don't think we need the second comma.

File: content/getting-started/performance.md [hunk 3]
```diff
@@ -40,28 +40,28 @@ them.
...
 Similar to MySQL, we merely retrieve data from commonly-available status views
-such as `pg_stat_activity` and similar. If running in "off-host" configuration,
+such as `pg_stat_activity` and similar. If running in [off-host](/getting-started/installing/#off-host-monitoring) configuration,
 where we cannot sniff TCP traffic, we measure query activity from `pg_stat_statements`,
 provided the extension is available.
 
 ## How Much Load Is Added To Redis?
 
-Virtually none. We basically ask for INFO and CONFIG GET * once a second, but that's pretty much it.
+Virtually none. We call `INFO` and `CONFIG GET *` once a second, and that's all.
```
Comment (Preetam) [inline content/getting-started/performance.md]: We only call...

File: content/getting-started/installing.md (hunk 1 above)
Comment (potocnyj) [inline content/getting-started/installing.md]: Dropped the LXC language here 👍

File: content/getting-started/installing.md (hunk 2 above)
Comment (potocnyj) [inline content/getting-started/installing.md]: Doh, removed.

File: content/getting-started/performance.md (hunk 3 above)
Comment (potocnyj) [inline content/getting-started/performance.md]: Done.

**Pull request merged**